### PR TITLE
Revert "Fix publish do revert (#2177)"

### PR DESCRIPTION
**Pull request by jonathanadler**
This reverts commit b1655e4b9d9c486cf122711483c77fb670fce282.

So I had to remove Janet in order to get the master build work again.
I wasn't able to find an easier solution.

It seems that the release script is running without node_modules and without the build result, so both `build-storybook` and `npx build-storybook` didn't work in this stage.
We had a similar issue when we moved to the monorepo (@argshook should probably remember, cause I negged him a lot about it 😛 ).

So I created this PR to try and bring Janet back.
We'll probably need to do some changes with this script, maybe even talk to CI.

btw, we can of course `npm i && npm run build` but it will take a long time, and this build already takes a long time.

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-core**

You can find it under one of the following URLs:
* https://wix-pages.com/wix-ui-core.pr_2178/

File: packages/wix-ui-core/package.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
     "storybook-start": "npm run storybook",
     "storybook:visual": "start-storybook -c ./.storybook/visual -p 6007",
     "start": "run-p yoshi-start storybook-start",
-    "release": "npx @wix/yoshi@4 release && npx teamcity-surge-autorelease@^1.0.0",
+    "release": "npx @wix/yoshi@4 release && npx build-storybook && npx janet deploy --ci",
```
Comment (argshook) [inline packages/wix-ui-core/package.json:68]:
```suggestion
    "release": "npx @wix/yoshi@4 release && npx build-storybook --package @storybook/react@5 && npx janet deploy --ci",
```

executing `npx build-storybook` would try to install `build-storybook`
package and run `build-storybook` from `node_modules/.bin`. But in this
case `build-storybook` is coming from `@storybook/react` package.

with `--package` flag we can set which package to get this binary from.

**Review by argshook — CHANGES_REQUESTED** [review]

**Comment by argshook** [comment]
we might need to add `npm install` to `release` script, cause i'm not sure if only `build-storybook` will be enough

File: packages/wix-ui-core/package.json (hunk 1 above)
Comment (jonathanadler) [reply packages/wix-ui-core/package.json:68]: Nice!

It's true, cause I tried it with `npx build-storybook` and it failed with
`npm ERR! 404 Not Found - GET http://npm.dev.wixpress.com/build-storybook - not_found`

**Comment by jonathanadler** [comment]
> we might need to add `npm install` to `release` script, cause i'm not sure if only `build-storybook` will be enough

@argshook `npm install` and `npm build` as well?

If I'm not mistaken, the release process does come with the `dist` folder, right?
Maybe we can have a storybook configuration that uses the `dist` components?
And then we won't need to `npm install` and `npm build`?

**Pull request closed without merging**